src/NIST-800-53/nist-800-53.ts

```diff
@@ -521,6 +542,74 @@ export class NIST80053Checks extends NagPack {
+        'Enable Amazon Relational Database Service (Amazon RDS) to help monitor Amazon RDS availability. This provides detailed visibility into the health of your Amazon RDS database instances.';
+      Annotations.of(node).addError(
+        this.createMessage(ruleId, info, explanation)
+      );
+    }
+    if (
+      !this.ignoreRule(
+        ignores,
+        'NIST.800.53-RDSInstanceDeletionProtectionEnabled'
+      ) &&
+      !nist80053RDSInstanceDeletionProtectionEnabled(node)
+    ) {
+      const ruleId = 'NIST.800.53-RDSInstanceDeletionProtectionEnabled';
+      const info =
+        'The RDS DB Instance does not have deletion protection enabled - (Control ID: SC-5).';
+      const explanation =
+        'Ensure Amazon Relational Database Service (Amazon RDS) instances have deletion protection enabled. Use deletion protection to prevent your Amazon RDS instances from being accidentally or maliciously deleted, which can lead to loss of availability for your applications.';
+      Annotations.of(node).addError(
+        this.createMessage(ruleId, info, explanation)
+      );
+    }
+    if (
+      !this.ignoreRule(ignores, 'NIST.800.53-RDSInstanceMultiAzSupport') &&
+      !nist80053RDSInstanceMultiAZSupport(node)
+    ) {
+      const ruleId = 'NIST.800.53-RDSInstanceMultiAzSupport';
+      const info =
+        'The RDS DB Instance does not have multi-AZ support - (Control IDs: CP-10, SC-5, SC-36).';
+      const explanation =
+        'Multi-AZ support in Amazon Relational Database Service (Amazon RDS) provides enhanced availability and durability for database instances. When you provision a Multi-AZ database instance, Amazon RDS automatically creates a primary database instance, and synchronously replicates the data to a standby instance in a different Availability Zone.';
+      Annotations.of(node).addError(
+        this.createMessage(ruleId, info, explanation)
+      );
+    }
+    if (
+      !this.ignoreRule(ignores, 'NIST.800.53-RDSInstancePublicAccess') &&
+      !nist80053RDSInstancePublicAccess(node)
+    ) {
+      const ruleId = 'NIST.800.53-RDSInstancePublicAccess';
+      const info =
+        'The RDS DB Instance allows public access - (Control IDs: AC-4, AC-6, AC-21(b), SC-7, SC-7(3)).';
+      const explanation =
+        'Amazon RDS database instances can contain sensitive information, and principles and access control is required for such accounts.';
+      Annotations.of(node).addError(
+        this.createMessage(ruleId, info, explanation)
+      );
+    }
+    if (
+      !this.ignoreRule(ignores, 'NIST.800.53-RDSStorageEncrypted') &&
+      !nist80053RDSStorageEncrypted(node)
+    ) {
+      const ruleId = 'NIST.800.53-RDSStorageEncrypted';
+      const info =
+        'RDS does not have storage encrypted - (Control IDs: SC-13, SC-28).';
+      const explanation =
+        'Because sensitive data can exist at rest in Amazon RDS instances, enable encryption at rest to help protect that data.';
```

Comment:
```suggestion
        'Because sensitive data can exist at rest in Amazon RDS instances and clusters, enable encryption at rest to help protect that data.';
```